Author: Florian Ragwitz <rafl@debian.org>
Description: Remove strange chars from pod, so pod2man produces sane output
Index: libxml-sax-writer-perl/lib/XML/SAX/Writer.pm
===================================================================
--- libxml-sax-writer-perl.orig/lib/XML/SAX/Writer.pm	2009-02-25 23:27:21.000000000 +0000
+++ libxml-sax-writer-perl/lib/XML/SAX/Writer.pm	2009-02-25 23:29:52.000000000 +0000
@@ -424,16 +424,16 @@
 
 =item * new(%hash)
 
-This is the constructor for this object.  It takes a number of
+This is the constructor for this object. It takes a number of
 parameters, all of which are optional.
 
-=item -- Output
+=item * -- Output
 
-This parameter can be one of several things.  If it is a simple
+This parameter can be one of several things. If it is a simple
 scalar, it is interpreted as a filename which will be opened for
-writing.  If it is a scalar reference, output will be appended to this
-scalar.  If it is an array reference, output will be pushed onto this
-array as it is generated.  If it is a filehandle, then output will be
+writing. If it is a scalar reference, output will be appended to this
+scalar. If it is an array reference, output will be pushed onto this
+array as it is generated. If it is a filehandle, then output will be
 sent to this filehandle.
 
 Finally, it is possible to pass an object for this parameter, in which
@@ -443,11 +443,11 @@
 
 If this parameter is not provided, then output is sent to STDOUT.
 
-=item -- Escape
+=item * -- Escape
 
 This should be a hash reference where the keys are characters
 sequences that should be escaped and the values are the escaped form
-of the sequence.  By default, this module will escape the ampersand
+of the sequence. By default, this module will escape the ampersand
 (&), less than (<), greater than (>), double quote ("), and apostrophe
 ('). Note that some browsers don't support the &apos; escape used for
 apostrophes so that you should be careful when outputting XHTML.
@@ -455,24 +455,24 @@
 If you only want to add entries to the Escape hash, you can first
 copy the contents of %XML::SAX::Writer::DEFAULT_ESCAPE.
 
-=item -- CommentEscape
+=item * -- CommentEscape
 
 Comment content often needs to be escaped differently from other
 content. This option works exactly as the previous one except that
 by default it only escapes the double dash (--) and that the contents
 can be copied from %XML::SAX::Writer::COMMENT_ESCAPE.
 
-=item -- EncodeFrom
+=item * -- EncodeFrom
 
 The character set encoding in which incoming data will be provided.
 This defaults to UTF-8, which works for US-ASCII as well.
 
-=item -- EncodeTo
+=item * -- EncodeTo
 
-The character set encoding in which output should be encoded.  Again,
+The character set encoding in which output should be encoded. Again,
 this defaults to UTF-8.
 
-=item -- QuoteCharacter
+=item * -- QuoteCharacter
 
 Set the character used to quote attributes. This defaults to single quotes (') 
 for backwards compatiblity.
